### ISSUE TITLE
Power BI не читается кириллица

### ISSUE THREAD

**Issue by pontid** [ClickHouse/clickhouse-odbc#257]
Доброго дня.
При подключении в power bi столбцы с кириллицей выдают "каракули". 
При попытке прописать в строке подключения driver={ClickHouse ODBC Driver (Unicode)} выкидывает ошибку таймаута.
Подскажите решение.
Win 10. Драйвер последний 64бит

**Comment by filimonov** [comment]
@pontid чем закончилась история?

**Comment by pontid** [comment]
@filimonov хотел сначала написать коммент, а потом стало стыдно на столь уважаемом ресурсе подобную ахинею писать)
Создал новый источник данных вида _ClickHouse (ClickHouse64) - оба варианта работают. И начала отображаться кириллица.

**Comment by pontid** [comment]
Когда выбираю в подключениях драйвер с Unicode скорость загрузки 800-1500 строк в секунду и на нем кириллица работает. Когда выбираю драйвер с ANSI скорость возрастает до 10-30к строк в секунду (в таблице 25кк строк), но кириллицу никак включить не получается. Подскажите что сделать чтобы и скорость была и кириллица была? =)

**Comment by savelov** [comment]
видимо надо вот эту оптимизацию сделать для лучшения производительности  - https://github.com/ClickHouse/clickhouse-odbc/issues/220

**Comment by traceon** [comment]
Да, будет исправлено в #220, однако ждать, что скорость с Unicode конвертациями будет такая же, как у ANSI, не стоит.

**Comment by traceon** [comment]
Можно еще поэкспериментировать с `CAP_ODBC_BIND_SUPPRESS_WIDE_CHAR`.
См. https://help.tableau.com/current/pro/desktop/en-us/odbc_capabilities.htm и https://github.com/ClickHouse/clickhouse-odbc/blob/master/packaging/clickhouse-odbc.tdc.sample

**Comment by Ev-bar** [comment]
Вопрос еще актуален, вы как нить решили проблему кроме как выбора Юникода?
